cli/config.go

```diff
@@ -155,11 +162,30 @@ func (c Config) FromYaml(r io.Reader) error {
 	c.UpdateDebug()
 	// Check app path validity
 	if (c[APPPATH].(string) != "" ) {
-		if _, err := os.Stat(c.AppPath()); errors.Is(err, os.ErrNotExist) {
-			// app does not exists, disable the app path and warn the user
-			fmt.Printf("warning - provided app path was not found : %s\n", c[APPPATH].(string))
-			c[APPPATH] = ""
+		apppath := c.AppPath()
+		switch runtime.GOOS {
+		case "windows":
+			for _, ext := range WIN_EXT {
+				if _, err := os.Stat(apppath + ext); !errors.Is(err, os.ErrNotExist) {
+					// found an executable with the given extension
+					c[APPPATH] = apppath + ext
+					break
+				}
+			}
```

Comment:
@NPavie Just checking: isn't this check already done within `AppPath()`?

Reply:
(will do it to while i'm at it just in case)